de.ifw_dresden.nagstamon.yml

```diff
@@ -0,0 +1,145 @@
+app-id: de.ifw_dresden.nagstamon
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: nagstamon
+finish-args:
+  - --socket=wayland  
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=~/.nagstamon:create
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: cpython
+    sources:
+      - type: archive
+        url: https://www.python.org/ftp/python/3.7.3/Python-3.7.3.tar.xz
+        sha256: da60b54064d4cfcd9c26576f6df2690e62085123826cff2e667e72a91952d318
+    build-options:
+      build-args:
+        - --share=network
```

Comment:
You can't use --share=network on Flathub. The builds must be reproducible.

Reply:
You can use https://github.com/flatpak/flatpak-builder-tools/tree/master/pip to generate dependencies automatically.

Reply:
I though "--share=network" was necessary. I just pushed some changes to the manifest removing that build arg and declaring the requirements by using the tool mentioned in the second comment.